python/dolfinx/fem/petsc.py

```diff
@@ -1118,3 +1126,94 @@ def interpolation_matrix(space0: _FunctionSpace, space1: _FunctionSpace) -> PETS
         Interpolation matrix.
     """
     return _interpolation_matrix(space0._cpp_object, space1._cpp_object)
+
+
+def copy_vec_to_function(
+    x: PETSc.Vec,
+    u: dolfinx.fem.Function,
+):  # type: ignore
+    """Update the solution for the unknown `u` with the values in `x`.
+
+    Args:
+        x: Data that is inserted into the solution vector.
+        u: Function data should be inserted into.
```

Comment:
```suggestion
        u: Function data should be coped to.
```
`u` should be a vector (see comment above)